index.html

```diff
@@ -1,17 +1,76 @@
-<!DOCTYPE html> 
+<!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8" />
-        <title>Leiden Software Developers</title>
-    </head>
-    <body>
-        <h1>Leiden Software Developers</h1>
-        <p>Future home of the Leiden Software Developers group.</p>
-        <p>Join us at:</p>
-        <ul>
-            <li><a href="https://www.meetup.com/leidendevs/">Meetup</a></li>
-            <li><a href="https://leidendevs.slack.com/">Slack</a></li>
-            <li><a href="https://old.reddit.com/r/LeidenDevs">Reddit</a></li>
-        </ul>
-    </body>
+
+<head>
+    <meta charset="utf-8" />
+    <title>Leiden Software Developers</title>
+    <style>
+        * {
+            font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, Ubuntu, roboto, noto, segoe ui, arial, sans-serif;
+            color: #111;
+        }
+
+        html {
+            font-size: 18px;
+        }
+
+        body {
+            padding: 4vw 10vw;
+        }
+
+        h1 {
+            font-weight: normal;
+        }
+
+        p {
+            line-height: 1.45;
+            max-width: 46ch;
+        }
+
+        img {
+            display: block;
+            width: 12ch;
+            margin-bottom: 3rem;
+            margin-left: -.25em;
+        }
+
+        a {
+            color: rgb(213, 12, 32);
+        }
+
+        footer {
+            margin-top: 4.35rem;
+            font-size: .875rem;
+        }
+
+        footer a {
+            color: currentColor
+        }
+
+        em {
```

Comment:
Try not to style purely based on structure. This is a very simple document so now it looks nice and clean, but for more complex documents this will be too generic and have unintended side effects.

Reply:
I consider this to be a throw away page rather than the basis of something longer lasting foundation, as a result I don't really think this issue as blocking. Let me know if you feel this _must_ be changed.